pkg/operator/operator.go

```diff
@@ -88,6 +105,16 @@ func (o *Operator) Run() error {
 		<-time.After(initRetryDelay)
 	}
 
+	// Run volume provisioner
+	// The controller needs to know what the server version is because out-of-tree
+	// provisioners aren't officially supported until 1.5
+	serverVersion, err := o.context.clientset.Discovery().ServerVersion()
+	if err != nil {
+		return fmt.Errorf("Error getting server version: %v", err)
+	}
+	pc := controller.NewProvisionController(o.context.clientset, resyncPeriod, provisionerName, o.volumeProvisioner, serverVersion.GitVersion, exponentialBackOffOnError, failedRetryThreshold, leasePeriod, renewDeadline, retryPeriod, termLimit)
```

Comment:
nit: this line is way too long, please break into multiple lines for readability.